src/plugins/intel_gpu/tests/test_cases/convolution_gpu_test.cpp

```diff
@@ -5381,12 +5406,19 @@ TEST(convolution_f32_fw_gpu, convolution_int8_b_fs_yx_fsv4_to_bfyx) {
     const int output_f = 12;
     const int input_f = 16;
     const int filter_xy = 5;
-    const int output_padding = 2;
     const int input_size_x = 1280;
     const int input_size_y = 720;
+    int output_padding = 2;
+    int input_padding = 2;
 
     auto& engine = get_test_engine();
 
+    if (engine.get_device_info().supports_immad) {
+        // Currently, oneDNN does NOT support input/output padding.
+        input_padding = 0;
+        output_padding = 0;
+    }
+
```

Comment:
the test name `fsv4_to_bfyx` suggests that its input is fsv4 and output is bfyx... If so, I think we can just disable this test for onednn as it is targeting for cldnn..

Reply:
Modified